### PR TITLE
#1986 fix indicator searchbar placeholder

### LINKED ISSUE #1974: Wrong placeholder for search bar in Indicators
## Describe the bug

Incorrect placeholder for Search bar use to filter Indicators (for HIV and REGIMEN in Supplier and Customer requisitions).

See here an example:

Screen Shot 2020-01-21 at 4 59 07 PM

### To reproduce

1. Go to Customer Requisitions
2. Open or create a new `HIV program` requisition
3. See error in search bar placeholder (as shown in photo above).

### Expected behaviour

It should say `Search by description or code`

### Proposed Solution

Change the search bar placeholder in each corresponding page. Probably need to add new localization strings to `generalStrings` in English and French. (As done in https://github.com/openmsupply/mobile/pull/1955)

### Version and device info

- App version: v3.2.0-rc3
- Tablet model: AVS
- OS version: Android 9.1

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #1974.

## Change summary

Updates searchbar placeholder text on indicators toggle.

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Supplier requisitions searchbar placeholder text updates on indicator toggle.
- [ ] Customer requisitions searchbar placeholder text updates on indicator toggle.

### Related areas to think about

If there are any general areas of the codebase your changes might have side affects on, mention them here

### CODE REVIEW

File: src/pages/CustomerRequisitionPage.js [hunk 1]
```diff
@@ -259,7 +267,7 @@ export const CustomerRequisition = ({
           <SearchBar
             onChangeText={onFilterData}
             value={searchTerm}
-            placeholder={`${generalStrings.search_by} ${generalStrings.item_name} ${generalStrings.or} ${generalStrings.item_code}`}
+            placeholder={`${generalStrings.search_by} ${placeholderStrings.name} ${generalStrings.or} ${placeholderStrings.code}`}
```
Comment (joshxg) [inline src/pages/CustomerRequisitionPage.js]: could get fancy with some tagging - poor example, and probably not worth it but fun!

```
const usingIndicators = true;
const getPlaceholder = (strings, v1, v2) =>
  `${v1 ? "indicator" : "Not indicator"}${strings[0]}${
    v2 ? "indicator" : "Not indicator"
  }`;

console.log(getPlaceholder`searchBy ${usingIndicators}${usingIndicators}`);
```

Comment (wlthomson) [reply src/pages/CustomerRequisitionPage.js]: > could get fancy with some tagging - poor example, and probably not worth it but fun!
> 
> ```
> const usingIndicators = true;
> const getPlaceholder = (strings, v1, v2) =>
>   `${v1 ? "indicator" : "Not indicator"}${strings[0]}${
>     v2 ? "indicator" : "Not indicator"
>   }`;
> 
> console.log(getPlaceholder`searchBy ${usingIndicators}${usingIndicators}`);
> ```

Does look fun :smiley:. Will put it on the post-CR "nice-to-haves" list!

File: src/localization/generalStrings.json [hunk 2]
```diff
@@ -40,6 +42,8 @@
     "requisition_number": "numéro de réquisition",
     "item_name": "nom de l'article",
     "item_code": "code de l'article",
+    "indicator_name": "description de l'indicateur",
+    "indicator_code": "code de l'indicateur",
```
[the remark this answers is not in the document]
Comment (wlthomson) [reply src/localization/generalStrings.json]: Good point, I think there are one or two missing translations. Will make an issue.

[the remark this answers is not in the document]
Comment (diegoevangelisti) [reply src/localization/generalStrings.json]: > Will make an issue.
Sounds like an order to yourself haha

**Pull request merged**